analyzer.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+# for details. All rights reserved. Use of this source code is governed by a
+# BSD-style license that can be found in the LICENSE file.
+
+runtime: custom
+env: flex
+service: analyzer
+
+resources:
+  cpu: 4
```

Comment:
These are kinda expensive to be running all the time.

Since we don't care about instant through-put, I'm wondering if using one CPU and setting the max instances higher would be a bitter idea.

@mkustermann ?